tests/integration/utils/testprogram.py

```diff
@@ -416,7 +419,10 @@ def detach_from_parent_group():
 
             popen_kwargs['preexec_fn'] = detach_from_parent_group
 
-        self.argv = [self.program]
+        if salt.utils.platform.is_windows():
```

Comment:
If you hit a very strange Windows machine, this might crash "program not found". Therefore I would also check `$PATH` if `python.exe` is also around.

Reply:
I messed up rebasing my commits on 2018.3 and accidentally pulled in a bunch of other commits. I'm going to try and fix that and force push the minimized changes back to this PR branch.